examples/proxy-example.php

```diff
@@ -0,0 +1,110 @@
+<?php
+declare(strict_types=1);
+
+require "vendor/autoload.php";
+
+use Momento\Auth\EnvMomentoTokenProvider;
+use Momento\Auth\EnvMomentoTokenProxyProvider;
+use Momento\Cache\SimpleCacheClient;
+use Momento\Config\Configurations\Laptop;
+use Monolog\Formatter\LineFormatter;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
+
+$CACHE_NAME = getenv("CACHE_NAME");
+if (!$CACHE_NAME) {
+    print "Error: Environment variable CACHE_NAME was not found.\n";
+    exit;
+}
+$ITEM_DEFAULT_TTL_SECONDS = 60;
+$KEY = "MyKey";
+$VALUE = "MyValue";
+
+// Setup
+$authProvider = new EnvMomentoTokenProxyProvider(
+    "MOMENTO_AUTH_TOKEN",
+    "momento-control:4443",
+    "momento-cache:4444"
+);
```

Comment:
There is so much other code in this example that this bit gets lost in the noise.  tbh I think if we want to have a proxy example it should probably just have the auth provider instantiation, with more comments about what it's doing and why, and then like one call to "listCaches" or something.  and we should remove the rest of the stuff that's duplicated from the other examples so that their attention is drawn to the important thing.